### PR TITLE
worker: create subtask for paused stage

### LINKED ISSUE #1090: support to query/operate paused subtask after re-scheduled
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

    1. `pause-task`
    2. stop the DM-worker instance to trigger re-schedule the source to another DM-worker instance
    3. `pause-task` again

2. What did you expect to see?

    `stage: paused`

3. What did you see instead?

    ```
    [2020/09/23 15:27:01.081 +08:00] [ERROR] [worker.go:418] ["fail to operate subtask stage"] [stage="{\"expect\":3,\"source\":\"mysql-replica-01\",\"task\":\"test\"}"] [error="[code=40054:class=dm-worker:scope=internal:level=high], Message: sub task with name test not found"] [errorVerbose="[code=40054:class=dm-worker:scope=internal:level=high], Message: sub task with name test not found\ngithub.com/pingcap/dm/pkg/terror.(*Error).Generate\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/pkg/terror/terror.go:263\ngithub.com/pingcap/dm/dm/worker.(*Worker).OperateSubTask\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/worker.go:272\ngithub.com/pingcap/dm/dm/worker.(*Worker).operateSubTaskStage\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/worker.go:458\ngithub.com/pingcap/dm/dm/worker.(*Worker).operateSubTaskStageWithoutConfig\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/worker.go:477\ngithub.com/pingcap/dm/dm/worker.(*Worker).handleSubTaskStage\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/worker.go:415\ngithub.com/pingcap/dm/dm/worker.(*Worker).observeSubtaskStage\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/worker.go:374\ngithub.com/pingcap/dm/dm/worker.(*Server).startWorker.func3\n\t/home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/server.go:822\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357"]
    ```

4. Versions of the cluster

    - DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):

        ```
        v2.0.0-rc.2 at least until https://github.com/pingcap/dm/commit/55fcd6db4446ef81e9081d932355b1e99a487604
        ```

[6889-77.log](https://github.com/pingcap/dm/files/5273347/6889-77.log)

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

close #1090 

### What is changed and how it works?

- create subtask even for paused stage task
- refine some logs

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported function/method change

Side effects

 - Increased code complexity

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Review by lance6716 — COMMENTED** [review]
LGTM

File: dm/worker/worker.go [hunk 1]
```diff
@@ -408,16 +410,19 @@ func (w *Worker) handleSubTaskStage(ctx context.Context, stageCh chan ha.Stage,
...
-		op = pb.TaskOp_Resume
-	case stage.Expect == pb.Stage_Paused:
-		op = pb.TaskOp_Pause
+		if stage.Expect == pb.Stage_Running {
+			op = pb.TaskOp_Resume
+		} else if stage.Expect == pb.Stage_Paused {
+			op = pb.TaskOp_Pause
+		}
 	case stage.IsDeleted:
 		op = pb.TaskOp_Stop
```
Comment (GMHDBJD) [inline dm/worker/worker.go:427]: Will stop-task report ErrWorkerSubTaskNotFound?

**Review by GMHDBJD — COMMENTED** [review]
rest LGTM

File: dm/worker/worker.go (hunk 1 above)
Comment (csuzhangxc) [reply dm/worker/worker.go:427]: Yes, for `stop-task`, we won't create/init one subtask and then stop it.

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**